docs/source/killed.rst

```diff
@@ -0,0 +1,105 @@
+.. _killed:
+
+Why did my worker die?
+----------------------
+
+A Dask worker can cease functioning for a number of reasons. These fall into the
+following categories:
+
+- the worker chooses to exit
+- an unrecoverable exception happens within the worker
+- the worker process is shut down by some external action
+
+Each of these cases will be described in more detail below. The *symptoms* you will
+experience when these things happen range from simply work not getting done anymore,
+to various exceptions appearing when you interact with your local client, such as
+``KilledWorker``, ``TimeoutError`` and ``CommClosedError``.
+
+Note the special case of ``KilledWorker``: this means that a particular task was
+tried on a worker, and it died, and then the same task was sent to another worker,
+which also died. After a configurable number of deaths, Dask decides to blame the
+task itself, and returns this exception.
+
+In every case, the first place to look for further information is the logs of
+the given worker, which may well give a complete description of what happened. These
+logs are printed by the worker to it's "standard output", which may be the text
```

Comment:
```suggestion
logs are printed by the worker to its "standard output", which may be the text
```

Also, does it go to stderr, typically? Maybe the distinction is not important here.